AdobeStockImageAdminUi/view/adminhtml/ui_component/media_gallery_listing.xml

```diff
@@ -9,12 +9,7 @@
          xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
     <settings>
         <buttons>
-            <button name="search_adobe_stock">
-                <param name="on_click" xsi:type="string">jQuery(".adobe-search-images-modal").trigger("openModal");</param>
-                <param name="sort_order" xsi:type="number">100</param>
-                <class>media-gallery-actions-buttons</class>
-                <label translate="true">Search Adobe Stock</label>
-            </button>
+            <button class="Magento\AdobeStockImageAdminUi\Ui\Component\Listing\SearchAdobeStockButton" name="search_adobe_stock" />
```

Comment:
I like the idea of the simplifying display button validation through the class of a button, but shouldn't we use modifiers for such case? @sivaschenko